src/test/java/adelaciobanu/Tests/Tema5/Herokuapp/BasicWebPageExample.java

```diff
@@ -0,0 +1,29 @@
+package adelaciobanu.Tests.Tema5.Herokuapp;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BasicWebPageExample {
+}
```

Comment:
Please remove this closing `}` as it's causing compilation error. Thanks!

Reply:
Only the closing `}` as it shouldn't be closed on that line but at the end of the class.